discord/guild.py

```diff
@@ -202,6 +202,7 @@ class Guild(Hashable):
         - ``ANIMATED_BANNER``: Guild can upload an animated banner.
         - ``ANIMATED_ICON``: Guild can upload an animated icon.
         - ``BANNER``: Guild can upload and use a banner. (i.e. :attr:`.banner`)
+        - ``CHANNEL_BANNER``: Guild can upload and use a channel banners.
```

Comment:
```suggestion
        - ``CHANNEL_BANNERS``: Guild can upload and use channel banners.
```
I suppose this is the correct form.

Reply:
You can't just change the name 😅

Reply:
My bad, I thought that would be the correct form. Just saw https://github.com/discord/discord-api-docs/pull/4068.